seacatauth/cookie/handler.py

```diff
@@ -436,7 +438,22 @@ async def _bouncer(self, request, parameters):
 
 
 	async def _authenticate_request(self, request, client_id=None):
-		return await self.CookieService.get_session_by_request_cookie(request, client_id)
+		cookie_value = self.CookieService.get_session_cookie_value(request, client_id)
+		if cookie_value is None:
+			return None
+
+		# First try if the cookie is a JWToken
+		try:
```

Comment:
Is there a fast way how to detect JWT token?

Reply:
we could use `if "." in cookie_value` before actually trying to parse the token. non-JWT values are just base64 strings so they should not contain dots.